etna/records/models.py

```diff
@@ -163,7 +163,7 @@ def has_url(self) -> bool:
             return True
 
     @cached_property
-    def title(self) -> str:
+    def summary_title(self) -> str:
```

Comment:
Is there also a 'title' field that we can pull out from somewhere?

Reply:
Yes, but needs approval https://national-archives.atlassian.net/browse/DF-417